.circleci/config.yml

```diff
@@ -1,53 +1,171 @@
+      - setup-automation
       - run:
-          name: install_write_behind_to_RedisGears_virtual_env
-          command: /var/opt/redislabs/modules/rg/python3_99.99.99/bin/python3 setup.py install
+          name: Setup Redis
+          command: ./deps/readies/bin/getredis --version 6
+      - setup-gears
       - run:
-          name: initialize_my_sql
+          name: Setup MySQL
           command: |
             service mysql start
-            mysql -u root --execute "CREATE DATABASE test"
-            mysql -u root --execute "CREATE TABLE test.persons (person_id VARCHAR(100) NOT NULL, first VARCHAR(100) NOT NULL, last VARCHAR(100) NOT NULL, age INT NOT NULL, PRIMARY KEY (person_id));"
-            mysql -u root --execute "INSERT INTO mysql.user (User,Host,authentication_string,ssl_cipher,x509_issuer,x509_subject) VALUES('demouser','localhost',PASSWORD('Password123!'),'','','');"
-            mysql -u root --execute "FLUSH PRIVILEGES;"
-            mysql -u root --execute "GRANT ALL PRIVILEGES ON test.* to demouser@localhost;"
-            mysql -u root --execute "FLUSH PRIVILEGES;"
+            ./sbin/setup-mysql
+      - run:
+          name: Install rgsync
+          command: /var/opt/redislabs/modules/rg/python3_99.99.99/bin/python3 setup.py install
+      - run:
+          name: Tests
+          command: python3 -m RLTest --module ../redisgears.so
+
+  platform-build:
+    parameters:
+      platform:
+        type: string
+    docker:
+      - image: 'debian:buster'
+    steps:
+      - platform-build-steps:
+          platform: <<parameters.platform>>
+ 
+  deploy-artifacts:
+    docker:
+      - image: 'cimg/base:2020.01'
+    steps:
+      - attach_workspace:
+          at: bin/artifacts
       - run:
-          name: install RLTest
-          command: pip install git+https://github.com/Grokzen/redis-py-cluster.git@master git+https://github.com/RedisLabsModules/RLTest.git sqlalchemy PyMySQL
+          name: Setup
+          command: sudo apt-get update && sudo apt-get install -y awscli
       - run:
-          name: run_tests
-          command: RLTest --module ../redisgears.so
+          name: Deploy Releases to S3
+          command: |
+            trim() {
+              local s2 s="$*"
+              until s2="${s#[[:space:]]}"; [ "$s2" = "$s" ]; do s="$s2"; done
+              until s2="${s%[[:space:]]}"; [ "$s2" = "$s" ]; do s="$s2"; done
+              echo "$s"
+            }
+            export AWS_ACCESS_KEY_ID=$(trim "$AWS_ACCESS_KEY_ID")
+            export AWS_SECRET_ACCESS_KEY=$(trim "$AWS_SECRET_ACCESS_KEY")
+            for f in bin/artifacts/*.zip; do
+              aws s3 cp $f s3://redismodules/rgsync/ --acl public-read
+            done
+
+platform-builds-requires: &platform-builds-requires
+  requires:
+    - build-centos7
+    - build-bionic
+    - build-xenial
+
+platform-builds-flow-defs: &platform-builds-flow-defs
+  requires:
+    - build_and_test
+  filters:
+    branches:
+      only: /.*/
```

Comment:
Should we make it on master and tags?

Reply:
Will go back to normal after testing is done, before we merge to master.